packages/ui-kit-components/src/components/SplashScreen/splashScreen.tsx

```diff
@@ -0,0 +1,70 @@
+import '@fontsource/merriweather/400.css';
+
+import styled from '@emotion/styled';
+import { Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import React, { useEffect, useState } from 'react';
+
+import LogoBrand from '../Branding/LogoBrand/LogoBrand';
+import LogoText from '../Branding/LogoText/LogoText';
+
+export default function SplashScreen() {
+  const [windowWidth, setWindowWidth] = useState(0);
+
+  useEffect(() => {
```

Comment:
This is well done (properly removing the listener on unmount). There is a react library called `react-use` which has a number of hooks I think including one to get screen width (or at least to use a media query), I am going to check tomorrow on standup if there is an appetite to introduce that library so we don't have to write out own hooks for common things like screen width.

Alternatively, you might want to write a custom hook `useWindowWidth` that can be shared across components.

Even though its only something we are doing once now, it is a good exercise to separate the screen width listener logic out from the component itself.